docs/reference/constant.md

```diff
@@ -130,12 +130,23 @@ The main advantage of doing this is its conciseness: the constant definition is
 line.  The built constant also has all of the multiplication and division operators types that
 `Constant` supports, as well as its perfect conversion policy to any `Quantity` type.
 
-The only disadvantage is the missing label, which will make printed quantities hard to understand
-because the constant will be represented as `[UNLABELED_UNIT]` in the compound label.
+The only disadvantage is the unit label for an ad hoc `Constant` will not be specific to that
+`Constant`.
```

Comment:
```suggestion
The only disadvantage is that an ad hoc `Constant` gets an ad hoc unit symbol, instead of a simple,
concise symbol.  The example below demonstrates the difference.
```

I thought this read a little more clearly.